sqlmesh/core/state_sync/base.py

```diff
@@ -14,9 +16,28 @@
     SnapshotNameVersionLike,
     SnapshotTableInfo,
 )
+from sqlmesh.utils import optional_import
 from sqlmesh.utils.date import TimeLike, now, to_datetime
 from sqlmesh.utils.errors import SQLMeshError
 
+MIGRATIONS = []
+MIGRATION_VERSION = 0
```

Comment:
Should we just call it `STATE_VERSION`? Here and the `_update_migration_version` definition.

Reply:
schema_version